mongodbatlas/resource_mongodbatlas_encryption_at_rest.go

```diff
@@ -177,7 +177,7 @@ func resourceMongoDBAtlasEncryptionAtRest() *schema.Resource {
 					Schema: map[string]*schema.Schema{
 						"enabled": {
 							Type:     schema.TypeBool,
-							Optional: true,
+							Required: true,
```

Comment:
[q] would this be a breaking change if I did not set it?

Reply:
I changed to Required because I re read the docs and the parameter enabled is required